docs/concepts/whatisvpp/index.rst

```diff
@@ -0,0 +1,22 @@
+.. _whatisvpp:
+
+=========================================
+What is VPP?
+=========================================
+
```

Comment:
I would put the context from whatisvpp (whatisvpp.rst) here and then have the detail in the menu.
I should need two clicks to get the high level in whatisvpp.rst.
Make sense?